packages/react/src/components/dateInput/DateInput.tsx

```diff
@@ -89,7 +90,7 @@ export const DateInput = React.forwardRef<HTMLInputElement, DateInputProps>(
      * Set the input value if value prop changes
      */
     useEffect(() => {
-      if (providedValue) {
+      if (didMount) {
         setInputValue(providedValue);
       }
     }, [providedValue]);
```

Comment:
The prevent "exhaustive-deps"-warning, we need to provide didMount in the dependency array.

Reply:
Adding `didMount`, as it was, to the array caused another issue; however, I changed the value of `didMount`, and now the warnings are gone.

Reply:
I ended up testing something similar 😁